tests/test_intl.py

```diff
@@ -1,12 +1,6 @@
-"""
-    test_intl
-    ~~~~~~~~~
-
-    Test message patching for internationalization purposes.  Runs the text
-    builder in the test root.
+"""Test message patching for internationalization purposes.
 
```

Comment:
This is also.

Reply:
I think this is fine -- the blank line is as there's a longer description on line 3:

https://github.com/sphinx-doc/sphinx/blob/f05d455313d2a521723f8f99f30b86a86fce1d4c/tests/test_intl.py#L1-L4

A

Reply:
Oops, you're right. withdrawing.